scs2-session-visualizer-jfx/src/main/java/us/ihmc/scs2/sessionVisualizer/jfx/session/log/VideoDataReader.java

```diff
@@ -1,351 +1,39 @@
 package us.ihmc.scs2.sessionVisualizer.jfx.session.log;
 
-import gnu.trove.list.array.TLongArrayList;
-import javafx.scene.image.WritableImage;
-import us.ihmc.codecs.demuxer.MP4VideoDemuxer;
-import us.ihmc.codecs.generated.YUVPicture;
-import us.ihmc.concurrent.ConcurrentCopier;
-import us.ihmc.euclid.tools.EuclidCoreTools;
 import us.ihmc.robotDataLogger.Camera;
 import us.ihmc.scs2.session.log.ProgressConsumer;
 
-import java.io.BufferedReader;
 import java.io.File;
-import java.io.FileNotFoundException;
-import java.io.FileReader;
 import java.io.IOException;
-import java.io.PrintWriter;
-import java.util.Arrays;
 
-public class VideoDataReader
+public interface VideoDataReader
```

Comment:
This is now an interface for the different VideoDataReaders that are implemented